frontend/packages/data-portal/app/components/Run/TomogramMetadataDrawer.tsx

```diff
@@ -194,7 +196,20 @@ export function TomogramMetadataDrawer() {
           },
           {
             label: t('canonicalStatus'),
-            values: [], // TODO
```

Comment:
might be good to go through the codebase and search `TODO(bchu)` or something to make sure there isn't something else I missed 😆